src/ansys/fluent/core/systemcoupling.py

```diff
@@ -73,19 +74,25 @@ def solve(self) -> None:
     def __get_syc_setup(self) -> dict:
         def get_scp_string() -> str:
             """Get SCP file contents in the form of the XML string."""
+
             fluent_cwd = self._solver.connection_properties.cortex_pwd
             scp_file_name = os.path.join(fluent_cwd, "fluent.scp")
             self._solver.setup.models.system_coupling.write_scp_file(
                 file_name=scp_file_name
             )
+            local_scp_file_name = scp_file_name
+            if self._solver.connection_properties.inside_container:
+                local_scp_file_name = os.path.join(os.getcwd(), "fluent.scp")
+                f.download(scp_file_name, local_scp_file_name)
```

Comment:
Shouldn't this be something like
```suggestion
                self._solver.download(scp_file_name, local_scp_file_name)
```
?